README.md

```diff
@@ -442,6 +442,367 @@ $params = array(
+        ->createPayment($external_id);
+```
+
+### 2.1 Get Transaction Status
+```php
+$statusByExternalIdArray = array(
+    "originalExternalId" => "uzi-order-testing66ce90ce90ee5",
+    "originalPartnerReferenceNo" => "uzi-order-testing66ce90ce90ee5",
+    "serviceCode" => "54",
+    "additionalInfo" => array()
+);
+
+$statusByReferenceArray = array(
+    "originalReferenceNo" => "A1202408280618283vcBaAmf7RID",
+    "serviceCode" => "54",
+    "additionalInfo" => array()
+);
+
+    /**
+     * Example code for getStatus using externalId
+     */
+    $snapBiResponse = SnapBi::transaction()
+        ->withBody($statusByExternalIdArray)
+        ->getStatus($external_id);
+        
+    /**
+     * Example code for getStatus using referenceNo
+     */
+    $snapBiResponse = SnapBi::transaction()
+        ->withBody($statusByReferenceArray)
+        ->getStatus($external_id);
+```
+
+### 2.1 Cancel Transaction
+
+```php
+$cancelByReferenceArray = array(
+    "originalReferenceNo" => "A120240902104935GBqSQK0gtQID"
+);
+        
+$cancelByExternalIdArray = array(
+    "originalExternalId" => "uzi-order-testing66d5983eabc71"
+);
+
+/**
+     * Basic implementation to cancel transaction using referenceNo
+     */
+    $snapBiResponse = SnapBi::transaction()
+        ->withBody($cancelByReferenceArray)
+        ->cancel($external_id);
+        
+ /**
+     * Basic implementation to cancel transaction using externalId
+     */
+    $snapBiResponse = SnapBi::transaction()
+        ->withBody($cancelByExternalIdArray)
+        ->cancel($external_id);
+
+```
+
+### 2.1 Refund Transaction
+
+```php
+$refundByExternalIdArray = array(
+    "originalExternalId" => "uzi-order-testing66cec41c7f905",
+    "partnerRefundNo" =>  "uzi-order-testing66cec41c7f905" . "refund-0001".rand(),
+    "reason" => "some-reason",
+    "additionalInfo" => array(),
+    "refundAmount" => array(
+        "value" => "100.00",
+        "currency" => "IDR"
+    ));
+
+$refundByReferenceArray = array(
+    "originalReferenceNo" => "A120240828062651Y0NQMbJkDOID",
+    "reason" => "some-reason",
+    "additionalInfo" => array(),
+    "refundAmount" => array(
+        "value" => "100.00",
+        "currency" => "IDR"
+    ));
+
+/**
+     * Example code for refund using externalId
+     */
+    $snapBiResponse = SnapBi::transaction()
+        ->withBody($refundByExternalIdArray)
+        ->refund($external_id);
+        
+/**
+     * Example code for refund using reference no
+     */
+    $snapBiResponse = SnapBi::transaction()
+        ->withBody($refundByReferenceArray)
+        ->refund($external_id);
+
+```
+
+### 2.1 Adding additional header / override the header
+
+you can add or override the header value, by utilizing the `->withAccessTokenHeader` or `withTransactionHeader`.
+You can refer to this docs to see the header value required by Snap-Bi https://docs.midtrans.com/reference/core-api-snap-open-api-overview, and see the default header on each payment method
```

Comment:
use link to text like [Snap-Bi](https://docs.midtrans.com/reference/core-api-snap-open-api-overview)